gravity/process_manager/supervisor_manager.py

```diff
@@ -287,6 +287,7 @@ def __update_service(self, config_file, config, attribs, service, instance_conf_
             "galaxy_infrastructure_url": attribs["galaxy_infrastructure_url"],
             "tusd": attribs["tusd"],
             "gx_it_proxy": attribs["gx-it-proxy"],
+            "proxy_path_prefix": attribs["proxy_path_prefix"],
```

Comment:
does this need a corresponding gravity/process_manager/systemd change?

Reply:
Ahh I thought that had been merged already, my mistake. Maybe @natefoo can provide pull this change in there eventually.

Reply:
Yeah I can handle merging forward to 1.x